static_src/test/functional/user_role.spec.js

```diff
@@ -0,0 +1,99 @@
+import http from 'http';
+import UserRoleElement from './pageobjects/user_role.element';
+
+describe('User roles', function () {
+  let cookieResult;
+
+  beforeEach(function () {
+    cookieResult = browser.getCookie('testing_user_role');
+    expect(cookieResult).toBeFalsy();
+  });
+
+  afterEach(function () {
+    browser.deleteCookie('testing_user_role');
+    cookieResult = browser.getCookie('testing_user_role');
+
+    expect(cookieResult).toBeFalsy();
+  });
+
+  describe('User role cookie test', function () {
+    describe('when cookie is set and deleted', function () {
+      it('should reflect the cookie content', function () {
+        browser.setCookie({ 'name': 'testing_user_role', 'value': 'space_manager_space_xx' });
+        browser.url('/uaa/userinfo');
+        cookieResult = browser.getCookie('testing_user_role').value;
+        expect(cookieResult).toBe('space_manager_space_xx');
+      });
+    });
+  });
+
+  describe('Org page', function () {
+    it('navigates to a org page', function () {
+      browser.url('/#/org/48b3f8a1-ffe7-4aa8-8e85-94768d6bd250');
+    });
+
+    it('has a title', function () {
+      expect(browser.getTitle()).toBe('cloud.gov dashboard');
+    });
+
+    it('has a page header', function () {
+      const pageHeader = browser.element('.test-page-header-title');
+      expect(pageHeader.getText()).toBe('fake-cf-deck-testing');
+    });
+
+    describe('org manager for org X then they should', function () {
+      it('be able to edit roles for org X', function () {
+        browser.setCookie({ 'name': 'testing_user_role', 'value': 'org_manager_org_x' });
```

Comment:
So here, it kinda sounds like you need to set the cooke, then reload or go the the org page. At that point you'll be in the correct state of the user being able to edit the roles for the org.

The user roles page element, should have an `isEditable` method which checks that the current UI is editable. So here you would select the correct user roles, and check that.